docker-compose-rule-junit4/src/main/java/com/palantir/docker/compose/events/EventConsumer.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.docker.compose.events;
+
+public interface EventConsumer {
```

Comment:
could we use `Consumer<Event>` instead?

Reply:
I actually like to keep this a concrete type for the API. It makes it easier to search for people using it on github, if we want to add more defaulted methods etc onto it in the future we can.